### PR TITLE
Better docs for Stripe.publishable_key and Stripe.endpoint_secret.

### PR DESCRIPTION
**Pull request by as1729**
#### What? Why?
The change adds some additional documentation to attr_accessor functions.

#### What should we test?
No testing should be required.

#### Release notes
Added developer documentation.

Changelog Category: Added

#### Documentation updates
None required as far as I am aware.

### CODE REVIEW

File: config/initializers/stripe.rb [hunk 1]
```diff
@@ -4,6 +4,8 @@
 # a bit cleaner than accessing keys in different ways.
 module Stripe
   class << self
+    # Returns the value of Stripe.publishable_key and Stripe.endpoint_secret.
+    # Attribute values can also be set by doing Stripe.publishable_key = <your_new_value>
```
Comment (sauloperez) [inline config/initializers/stripe.rb]: I think it would be worth also to mention `Stripe.endpoint_secret = <your_new_value>`

**Comment by sauloperez** [comment]
Thanks a lot for your contribution @as1729 and welcome to the Open Food Network! Don't hesitate to pick any [good first issue](https://github.com/openfoodfoundation/openfoodnetwork/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+good+label%3A%22good+first+issue%22) and join us on [Slack](https://openfoodnetwork.org/slack-invite). Besides, in the [README](https://github.com/openfoodfoundation/openfoodnetwork/blob/master/README.md) file you'll find everything you need to know.

File: config/initializers/stripe.rb (hunk 1 above)
Comment (sigmundpetersen) [reply config/initializers/stripe.rb]: Would you like to address this comment @as1729? Then it can be approved and merged

**Comment by mkllnk** [comment]
@sauloperez Are you happy to approve this PR? @as1729 doesn't seem to be active any more.

**Comment by sauloperez** [comment]
Yes, let's move it forward.

**Pull request merged**